packages/st2mistral/debian/rules

```diff
@@ -33,5 +33,5 @@ override_dh_installdeb:
 	dh_installdeb
 
 override_dh_virtualenv:
-	dh_virtualenv --extra-pip-arg '--find-links=$(WHEELDIR)' \
+	dh_virtualenv --extra-pip-arg '--find-links=$(WHEELDIR)' --preinstall psycopg2 \
```

Comment:
@DoriftoShoes better you put it here https://github.com/StackStorm/st2-packages/blob/master/packages/st2mistral/Makefile#L9-L16, so that we don't scatter everything over different places.